models/chunk_channels.go

```diff
@@ -79,6 +79,21 @@ func GetReadyChannels() ([]ChunkChannel, error) {
 	return channels, err
 }
 
+// GetReadyChannels grabs one ready channel
+func GetOneReadyChannel() (ChunkChannel, error) {
+
+	channel := ChunkChannel{}
+
+	err := DB.RawQuery("SELECT * from chunk_channels WHERE "+
+		"est_ready_time <= ? ORDER BY est_ready_time;", time.Now()).First(&channel)
```

Comment:
When is `est_ready_time` updated? Might want to atomically update it here (rename the method to `GetAndReserveChannel`) because concurrent threads might get the same channel from the DB.

Doesn't have to be this PR, but we should make an issue to get back to this if it becomes a problem.

Reply:
It gets updated in a method called SetEstimatedReadyTime in iota_wrappers.go  

Eventually I want to relocate it to the chunk_channels.go file but there was some kind of import cycle when I tried that, so for now just keeping it in the iota_wrappers file.